### PR TITLE
TINY-6018: Fixed tooltipping throwing an exception when used with receiving

### PR DESCRIPTION
**Pull request by lnewson**

### CODE REVIEW

File: modules/tinymce/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Version 5.3.1 (TBD)
+    Fixed the autocompleter throwing an exception when tooltipping was configured #TINY-6018
```
Comment (metricjs) [inline modules/tinymce/changelog.txt:2]: this maybe makes more sense as "tooltips" than "tooltipping" to customers?

Comment (lnewson) [reply modules/tinymce/changelog.txt:2]: It's not actually "tooltips" though, it's more of a popover that's currently undocumented and only used by mentions.

Comment (metricjs) [reply modules/tinymce/changelog.txt:2]: that's not confusingly named at all XD I guess if we keep it as tooltipping, we could just add speech marks to show that it's a term? I can't think of any good alternative words.

Comment (lnewson) [reply modules/tinymce/changelog.txt]: Dropping this entry since the feature isn't documented as discussed on slack.

**Pull request merged**